nncf/common/graph/graph_matching.py

```diff
@@ -61,18 +67,26 @@ def _find_all_matches(self, nodes, graph):
                 all_matches.append((n, following))
         return all_matches
 
-    def all_matches(self, nodes, graph):
+    def all_matches(self, nodes: List[Dict], graph: nx.DiGraph):
         return self._find_all_matches(nodes, graph)
 
-    def match(self, nodes, graph):
+    def match(self, nodes: List[Dict], graph: nx.DiGraph):
         all_matches = self._find_all_matches(nodes, graph)
         if not all_matches:
             return None, None
         return max(all_matches, key=lambda x: len(x[0]))
 
 
+
 class ConcatExpression(Expression):
-    def __init__(self, expressions):
+    """A composite expression that matches to a node path in the graph which is a concatenation
```

Comment:
Please, format the docstring according to the Style Guide.

Reply:
Done